AWSClientRuntime/Sources/Config/BundleConfiguration.swift

```diff
@@ -0,0 +1,30 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
```

Comment:
question. can we be specific here with our import? like `import struct Foundation.Bundle`?

Reply:
I tried that, and it didn't work.  I can try again.

Reply:
works:
```
import Foundation.NSBundle
```